src/relay/op/tensor/unary.cc

```diff
@@ -78,5 +78,37 @@ RELAY_REGISTER_OP("concat")
 .set_support_level(1)
 .add_type_rel("Concat", ConcatRel);
 
+// Clip
+struct ClipAttrs : public tvm::AttrsNode<ClipAttrs> {
+  // TODO: should I enforce these be 0 dim?
+  Constant a_min;
```

Comment:
just use double as they are constant and double is enough to represent most numbers we care about